result/lldpneighbors.yaml

```diff
@@ -0,0 +1,155 @@
+components:
+  schemas:
+    LldpNeighbors.State.Request:
+      description: >-
+        The request to retrieve LLDP per interface neighbor information.
```

Comment:
"The request to retrieve LLDP neighbor information for a given interface."

Reply:
updated